File: tundra-cli/benchmarks/iterative-fibonacci/fib.java
public class fib {
  public static void main(String[] args) {
    int n = 80;
    long a = 0, b = 1;
    for (int i = 0; i < n; i++) {
      long tmp = a + b;
      a = b;
      b = tmp;
    }
    System.out.println(a);
  }
}

File: tundra-cli/benchmarks/iterative-fibonacci/fib.zig
const std = @import("std");
pub fn main() void {
    var a: usize = 0;
    var b: usize = 1;
    const n = 80;
    for (0..n) | _ | {
        const tmp = a + b;
        a = b;
        b = tmp;
    }
    std.debug.print("{}\n", .{a});
}